intermediate_source/nvfuser_intro_tutorial.py

```diff
@@ -0,0 +1,679 @@
+# -*- coding: utf-8 -*-
+"""
+Getting Started - Accelerate Your Scripts with nvFuser
+****************************
+
+**Authors**: `Christian Sarofeen <https://github.com/csarofeen>`_
+`Piotr Bialecki <https://github.com/ptrblck>`_
+`Kevin Stephano <https://github.com/kevinstephano>`_
+`Jie Jiang <https://github.com/jjsjann123>`_
+`Masaki Kozuki <https://github.com/crcrpar>`_
+`Neal Vaidya`
+
+
+Introduction
+------------
+
+This tutorial will demonstrate how you can accelerate your networks
+with nvFuser. nvFuser is a Deep Learning Compiler that just-in-time
+compiles fast and flexible GPU specific code to reliably accelerate
+users' networks automatically, providing speedups for deep learning
+networks running on Volta and later CUDA accelerators by generating
+fast custom “fusion” kernels at runtime. nvFuser is specifically
+designed to meet the unique requirements of the PyTorch community,
+and it supports diverse network architectures and programs with
+dynamic inputs of varying shapes and strides. If you’re unfamiliar
+with nvFuser please review [this blog post, INSERT BLOG POST LINK].
+
+Importing Packages and Selecting a Device
+-----------------------------------------
+In order to run this tutorial and see the benefits of using nvfuser,
+you would need to install the `1.12.0` PyTorch release as well as
+`functorch` `0.2` or newer version of them.
```

Comment:
I installed functorch using `pip install --user "git+https://github.com/pytorch/functorch.git"` and it seems it doesn't include `networkx` needed for `memory_efficient_fusion` 
```
RuntimeError: Need networkx installed to perform smart recomputation heuristics
```
I think the dependency on `networkx` should be included here.